### PR TITLE
142 add more localization

### PR DESCRIPTION
**Pull request by MatyashDen**

### CODE REVIEW

**Comment by MatyashDen** [comment]
#250😅

File: ios/Codeforces Watcher/Application/core/extensions/UIColor+Extension.swift [hunk 1]
```diff
@@ -31,23 +31,23 @@ func colorTextByUserRank(text: String, rank: String?) -> NSMutableAttributedStri
     switch (rank) {
     case nil:
         color = Palette.black
-    case "newbie":
+    case "newbie", "новичок":
```
Comment (yevhenii-kanivets) [inline ios/Codeforces Watcher/Application/core/extensions/UIColor+Extension.swift:34]: Wow, should we also do it for Android?

**Comment by MatyashDen** [comment]
> @MatyashDen good job 👍 I've not checked every translation in a hope you copy/pasted it exactly from Android, right?

@yevhenii-kanivets yes, i copy/pasted

File: ios/Codeforces Watcher/Application/core/extensions/UIColor+Extension.swift (hunk 1 above)
Comment (MatyashDen) [reply ios/Codeforces Watcher/Application/core/extensions/UIColor+Extension.swift:34]: Yes, we should

**Pull request merged**